src/cdk/tree/nested-node.ts

```diff
@@ -62,17 +64,21 @@ export class CdkNestedTreeNode<T> extends CdkTreeNode<T> implements AfterContent
   }
 
   ngAfterContentInit() {
-    this._tree.treeControl.getChildren(this.data).pipe(takeUntil(this._destroyed))
-        .subscribe(result => {
-          if (result && result.length) {
-            // In case when nodeOutlet is not in the DOM when children changes, save it in the node
-            // and add to nodeOutlet when it's available.
-            this._children = result as T[];
-            this._addChildrenNodes();
-          }
-        });
+    const getChildren: ((data: T) => Observable<T[]>) | null = this._tree.treeControl.getChildren;
```

Comment:
If `treeControl.getChildren` depends on any of the members of `treeControl` this will fail because saving a reference to a function like this loses the `this` context. You could use `bind`, but it seems like it would be more straightforward to just reference it from `treeControl`

(same for other places using this pattern)